src/linagora.esn.unifiedinbox/app/components/composer/composer.controller.spec.js

```diff
@@ -186,6 +186,20 @@ describe('The inboxComposerController controller', function() {
       expect(ctrl.onTitleUpdate).to.have.been.calledWith({ $title: 'subject' });
     });
 
+    it('should remove duplicate email adresses in the To property', () => {
```

Comment:
Should also have a test when there are multiple email values, not only the same as this one